homeassistant/components/sensehat/sensor.py

```diff
@@ -43,8 +43,8 @@
 
 def get_cpu_temp():
     """Get CPU temperature."""
-    t_cpu = os.popen("cat /sys/class/thermal/thermal_zone0/temp").readline()
-    return float(t_cpu)
+    t_cpu = Path("/sys/class/thermal/thermal_zone0/temp").read_text().replace("\n", "")
```

Comment:
```suggestion
    t_cpu = Path("/sys/class/thermal/thermal_zone0/temp").read_text().stripe()
```

Reply:
Yep, I thought about that :) but I followed previous grammar. Thanks